lib/perl/Genome/Model/Build/ClinSeq.pm

```diff
@@ -29,4 +29,59 @@ sub should_run_exome_cnv {
     return ($self->exome_build and $self->processing_profile->exome_cnv);
 }
 
+sub input_builds {
+    my $self = shift;
+
+    my @input_builds = ( $self->tumor_rnaseq_build, $self->normal_rnaseq_build );
+    for my $accessor (qw(wgs_build exome_build)) {
+        if ($self->$accessor) {
+            push @input_builds, $self->$accessor;
+            for my $build (qw(tumor_build normal_build)) {
+                if ($self->$accessor->can($build)) {
```

Comment:
This could also be:
```perl
if (my $input_build = $self->$accessor) {
   push @input_builds, $input_build;
   for my $build (qw(tumor_build normal_build)) {
       if ($input_build->can($build)) {
```